_includes/sections/hosting-provider.html

```diff
@@ -1,5 +1,9 @@
 <h1 id="host" class="anchor"><a href="#host"><i class="fas fa-link anchor-icon"></i></a> Secure Hosting Provider</h1>
 
+<div class="alert alert-warning" role="alert">
+  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please verify their privacy policies.</strong>
```

Comment:
```suggestion
  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please read their privacy policies.</strong>
```

Lazy suggestion, so I don't have to terminal it.